CONTRIBUTING.md

```diff
@@ -1,67 +1,67 @@
-# Welcome to the gematik contributing guide <!-- omit in toc -->
- 
-Thank you for investing your time in contributing to our projects!
-
-Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
```

Comment:
In diesem Projekt haben wir kein Contributing, oder? 
Also kann das doch dann auch raus und wenn es noch in der Readme oder so steht, kann das auch gelöscht werden.